templates/docs/troubleshooting.md

```diff
@@ -10,3 +10,14 @@ $ go get -u github.com/motemen/gore
 
 For information see [https://github.com/gobuffalo/buffalo/issues/108](https://github.com/gobuffalo/buffalo/issues/108) and [https://github.com/motemen/gore/issues/63](https://github.com/motemen/gore/issues/63).
 {{/panel}}
+
+{{#panel title="buffalo dev fails to start"}}
```

Comment:
You should pass a name argument to the panel. The default is to build an anchor tag from the title, but it would be crazy long. Use the name argument to specify a shorter one.